### DISCUSSION TITLE
Support for windows

### DISCUSSION THREAD

**Discussion by Ninja4Code** [containerd/nerdctl#2224] [Q&A]
Basically can I run the major commands (build, run, start stop) on Windows?  The documentation talks about experimental features but doesn't really get into the detail.  I'm not expecting the full features of Docker EE.

**Comment by AkihiroSuda** [comment]
Build doesn't work on Windows IIUC

**Comment by Ninja4Code** [comment]
That's kind of a core container feature there that's missing ;)  Could someone use containerd for the build?  Is there a roadmap to support Windows containers?  Like many things, it's about resources, so do you have anyone working on Windows features? Are you open to someone jumping in and working on those aspects? 
I'm asking these questions because Microsoft no longer supports Docker EE.  Mirantis wants to charge almost a thousand dollars a year for support, which I think is ridiculous.  There's the Moby project that doesn't do a good job of documentation.  The API is solid.
